inc/abstractquestionparameter.class.php

```diff
@@ -65,7 +65,7 @@ abstract class PluginFormcreatorAbstractQuestionParameter
     *                - fieldName: name of the HTML input tag
     *                - label    : label for the parameter
     */
-   public function __construct(PluginFormcreatorFieldInterface $field, array $options) {
+   public function setField(PluginFormcreatorFieldInterface $field, array $options) {
```

Comment:
The signature of this method is diferent from the signature in the parent classes. PHP now complains, and this needs a fix.